pom.xml

```diff
@@ -165,6 +160,12 @@
             <artifactId>mock-javamail</artifactId>
             <version>2.0</version>
             <scope>test</scope>
+            <exclusions>
```

Comment:
Nit: A note explaining why would be interesting

Reply:
This is a pretty standard exclusion used in various plugins IIRC. I am a bit hesitant to leave comments with too many details with such cases because if you doubt that an `exclusion` is still actually necessary, it is best to just try deleting it and running

```bash
mvn clean verify -Dtest=InjectedTest
```